setup.py

```diff
@@ -228,7 +228,7 @@ def make_release_tree(self, base_dir, files):
             """driver. Please use it only when a kernel driver is already present on the\n""" \
             """system. Otherwise, please install chipsec from source, using the following\n""" \
             """procedure:\n""" \
-            """  https://github.com/chipsec/chipsec/blob/master/chipsec-manual.pdf"""
+            """  https://github.com/chipsec/chipsec/blob/main/chipsec-manual.pdf"""
```

Comment:
Maybe add this line:
`"""  https://chipsec.github.io/"""`
(and add a `\` at the end of this line.